py/server/tests/test_parquet.py

```diff
@@ -576,6 +581,85 @@ def test_read_parquet_from_s3(self):
             read("s3://dh-s3-parquet-test1/multiColFile.parquet", special_instructions=s3_instructions).select()
         # TODO(deephaven-core#5064): Add support for local S3 testing
 
+    def verify_metadata_files(self, root_dir):
+        metadata_file_path = os.path.join(root_dir, '_metadata')
+        self.assertTrue(os.path.exists(metadata_file_path))
+        common_metadata_file_path = os.path.join(root_dir, '_common_metadata')
+        self.assertTrue(os.path.exists(common_metadata_file_path))
+
+    def test_writing_partitioned_data(self):
```

Comment:
```suggestion
    def test_write_partitioned_data(self):
```
Just made the naming consistent with the other tests -- I think.